app/navigation/FundRaisingNavigator.scala

```diff
@@ -131,18 +130,27 @@ class FundRaisingNavigator @Inject()(implicit frontendAppConfig: FrontendAppConf
   private val checkRouteMap: Page => UserAnswers => Call = {
 
     case FundRaisingPage => userAnswers: UserAnswers => userAnswers.get(FundRaisingPage) match {
-      case Some(items) if items.toSeq.equals(Seq(Other)) =>  routes.DeadEndController.onPageLoad()
+      case Some(items) if items.toSeq.contains(Other) => userAnswers.get(OtherFundRaisingPage) match {
+        case Some(_) => operationFundsRoutes.OperationsFundsSummaryController.onPageLoad()
+        case _ => operationFundsRoutes.OtherFundRaisingController.onPageLoad(CheckMode)
```

Comment:
do we need line 135 if we already tested if it contains other

Reply:
yes, we do:

if a user answers `Other` to this question, and previously they had also answered `Other`, they will get sent to the summary.
if a user answers `Other` to this question, but previously they didn't have `Other` selected, they will get sent to the OtherFundRaisingPage. 

(the capitalisation of 'FundRaising' is still bothering me)